changelog/range-error.dd

```diff
@@ -0,0 +1,6 @@
+RangeError now has bounds information
+
+Errors resulting from bad indexing will now contain the length of 
+the array that was sliced as well as the offending index for `arr[bad]` 
+or for bad slices `arr[bad1 .. bad2]`. The associated runtime functions 
+`onRangeError` and `_d_arraybounds[p]` have changed signature to match.
```

Comment:
Have a look at https://github.com/dlang/druntime/pull/2413 for an example of nice error messages.

I would do the following:
- simple example of mis-indexing
- use the `$(CONSOLE)` macro to show this new feature of